### PR TITLE
Reorder ldflags

### PR DESCRIPTION
**Pull request by hiraditya**
w.r.t. issue#1133 [mxnet] Unable to compile c++ exaxmple mlp.cpp
https://github.com/dmlc/mxnet/issues/1133

### CODE REVIEW

File: example/cpp/Makefile [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 
+CC=g++
+
 mlp: ./mlp.cpp
-	g++ -std=c++0x $(CFLAGS) $(LDFLAGS) -o $@ $^
+	$(CC) -std=c++0x $(CFLAGS) -o $@ $^ $(LDFLAGS)
 
 use_ndarray: ./use_ndarray.cpp
-	g++ -std=c++0x $(CFLAGS) $(LDFLAGS) -o $@ $^
+	$(CC) -std=c++0x $(CFLAGS) -o $@ $^ $(LDFLAGS)
```
Comment (tqchen) [inline example/cpp/Makefile]: change to $(CXX)

**Comment by winstywang** [comment]
@hiraditya Could you update your branch so that we can merge in this PR?

**Comment by hiraditya** [comment]
Done!

**Comment by tqchen** [comment]
Have added you to https://github.com/dmlc/mxnet/blob/master/CONTRIBUTORS.md

Thanks for the contribution and make mxnet more accessible for all users!

**Pull request merged**